Gemfile

```diff
@@ -163,6 +163,7 @@ group :test do
   gem 'simplecov', require: false
   gem 'test-prof'
   gem 'webmock'
+  gem 'vcr'
```

Comment:
This should be alphabetical.
```suggestion
  gem 'vcr'
  gem 'webmock'
```

Reply:
Thanks, was unaware of this convention - done and squashed with the first commit.